content/200-concepts/100-components/02-prisma-client/080-null-and-undefined.mdx

```diff
@@ -12,90 +12,235 @@ Prisma Client differentiates between `null` and `undefined`:
 - `null` is a **value**
 - `undefined` means **do nothing**
 
-> **Note**: This is particularly important to account for in [a **Prisma with GraphQL context**, where `null` and `undefined` are interchangeable](#use-case-null-and-undefined-in-a-graphql-resolver).
+<Admonition type="info">
 
-In the following example, if `emailInput` is `null`, the query sets `email` (a **mandatory** field) to `undefined` - which means ✔ **do not include this in the update**:
+This is particularly important to account for in [a **Prisma with GraphQL context**, where `null` and `undefined` are interchangeable](#use-case-null-and-undefined-in-a-graphql-resolver).
+
+</Admonition>
+
+The data below represents a `User` table. This set of data will be used in all of the examples below:
+
+| id  | name    | email             |
+| --- | ------- | ----------------- |
+| 1   | Nikolas | nikolas@gmail.com |
+| 2   | Martin  | martin@gmail.com  |
+| 3   | _empty_ | sabin@gmail.com   |
+| 4   | Tyler   | tyler@gmail.com   |
+
+</TopBlock>
+
+<!-- TODO Simple example with `findMany` and `findFirst` that shows how a query with `foo: undefined` (e.g. https://github.com/prisma/prisma/issues/17723) returns everything or any first entry as it is filtered out of the query -->
```

Comment:
Is this a todo for later or for this PR?

Reply:
No idea, you'll have to figure that out from the context.